lib/src/Pos/Diffusion/Full/Ssc.hs

```diff
@@ -16,6 +16,9 @@ import           Pos.Binary.Class (Bi)
 import           Pos.Binary.Crypto ()
 import           Pos.Binary.Infra ()
 import           Pos.Binary.Ssc ()
+-- Message instances for various types.
+-- TODO should move these into the Diffusion module subtree.
```

Comment:
Will it be done in this PR or elsewhere?

Reply:
Elsewhere